curriculum/challenges/english/02-javascript-algorithms-and-data-structures/intermediate-algorithm-scripting/dna-pairing.md

```diff
@@ -10,7 +10,7 @@ dashedName: dna-pairing
 
 The DNA strand is missing the pairing element. Take each character, get its pair, and return the results as a 2d array.
 
-[Base pairs](http://en.wikipedia.org/wiki/Base_pair) are a pair of AT and CG. Match the missing element to the provided character.
+Base pairs are a pair of AT and CG, which form building blocks of the DNA double helix. Match the missing element to the provided character.
 
 Return the provided character as the first element in each array.
 
```

Comment:
I feel like we could perhaps make a more comprehensive edit here. Perhaps reordering:

Pairs of DNA strands consist of protein base pairs. Base pairs are represented by the characters AT and CG, which form building blocks of the DNA double helix.

The DNA strand is missing the pairing element. Write a function to match the missing base pairs for the provided DNA strand. For each character in the provided string, find the base pair character. Return the results as a 2d array.

Reply:
I do like it. I added wrapped AT and CG in <em>, because I thought they are important as well here is how it looks like 
![image](https://user-images.githubusercontent.com/88248797/170306337-7a5083ab-cbc3-4360-877a-4b6f4e938180.png)

Reply:
I like the change. Thanks!